core/hash.rbs

```diff
@@ -391,7 +391,7 @@ class Hash[unchecked out K, unchecked out V] < Object
   #     a is 100
   #     b is 200
   #
-  def each: () { ([ K, V ] arg0) -> untyped } -> self
+  def each: () { (K, V) -> untyped } -> self
```

Comment:
Passing a tuple is intentional and I believe it's consistent with the implementation.

```
irb(main):001:0> RUBY_VERSION
=> "3.0.0"
irb(main):002:0> {a: 1}.each.to_a
=> [[:a, 1]]
```

Reply:
Then maybe it's a problem of `steep` that it doesn't acknonwledge these spreads, is that it? Probably related to https://github.com/ruby/rbs/issues/734#issuecomment-898276685